src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int defaultCircleRadius = 10;
+    private final String defaultCircleColor = "white";
+    private final int figureCount = 5;
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int num = random.nextInt(figureCount);
+
+        switch (num) {
+            case 0:
+                return new Square(random.nextInt(100));
```

Comment:
create private methods for preparing Figures
```suggestion
                return prepareSquare();
```